src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -22,15 +26,23 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a loan to the loan book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
+            + PREFIX_NRIC + "NRIC "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_BIKE + "BIKE "
+            + PREFIX_LOANRATE + "LOANRATE "
+            + PREFIX_LOANTIME + "LOANTIME "
```

Comment:
Not correct. Loan start time is NOT a parameter when adding a loan!
The start time based on when the loan was created.